### PR TITLE
Single article

### PR DESCRIPTION
**Pull request by mckhannah1901**
### JIRA
ATH-954: View Single Blog Article

### What is being Done
- Anyone can view a blog post, regardless of being logged in or not
- Each article can be selected form the list of articles and viewed on its own page

### How to Verify
1. Check out this PR
2. Without logging in, select to view blogs from the landing page
3. When on the page that shows the list of blog articles, select show on the post you'd like to view
4. The selected blog post will be loaded onto its own page

### CODE REVIEW

File: landing_page/app/views/microposts/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<p>
```
Comment (jamesfairbairn-wpengine) [inline landing_page/app/views/microposts/show.html.erb]: A `<div>` tag may be more appropriate here. I believe `<p>` tags are mainly supposed to be used for paragraphs of text, not to contain headers.

**Comment by alvarocavalcanti** [comment]
The PR title should be more descriptive of what these changes are.

Also, the description should have a link to the Jira issue as well as more details of what was implemented. For reference, see the pull requests below:

- https://github.com/wpengine/wpe-wp-sign-on-plugin/pull/119
- https://github.com/wpengine/my_wpe/pull/5865
- https://github.com/wpengine/my_wpe/pull/5839

File: landing_page/app/views/microposts/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
```
Comment (alvarocavalcanti) [inline landing_page/app/views/microposts/show.html.erb]: You have been using only `<p>` here, which isn't recommended as per HTML standards. Have a look at these:

- [HTML Sections](https://html.spec.whatwg.org/multipage/sections.html#sections), and how they should be used to categorize content
- [HTML Grouping Content](https://html.spec.whatwg.org/multipage/grouping-content.html#grouping-content)
- [HTML Text Level Semantics](https://html.spec.whatwg.org/multipage/text-level-semantics.html#text-level-semantics)

File: landing_page/app/views/microposts/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<p>
+    <strong>Updated at:</strong>
+    <%= @micropost.updated_at %>
+</p>
+
+<br></br>
+<br></br>
+<br></br>
+
+<h6><%= link_to "Back to List of Blog Posts", microposts_path %></h6>
```
Comment (alvarocavalcanti) [inline landing_page/app/views/microposts/show.html.erb]: Here you used `h6` but you don't have any other headings in the page. Headers are used not only for visuals, as in bigger or smaller texts, but also to categorize the contents of the page. Ideally you should always start with a `h1` and continue on organizing the content. Have a look at this article: https://www.w3schools.com/html/html_headings.asp

**Comment by mckhannah1901** [comment]
Thanks for the feedback @alvarocavalcanti , I'll definitely keep this in mind in future! :)

**Pull request merged**